media/css/firefox/windows-64-bit.scss

```diff
@@ -0,0 +1,76 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../protocol/css/includes/lib';
+@import '../../protocol/css/components/article';
+@import '../../protocol/css/components/call-out';
+
+h1, h2 {
```

Comment:
Nit: rather than changing heading styles using a global scope, I'd suggest applying this only to the components that need the styling.